### ISSUE TITLE
Improve `var_names` behavior

### ISSUE THREAD

**Issue by ColCarroll** [arviz-devs/arviz#325]
If var_names is a string: interpret it as a list of just that string

**Comment by ahartikainen** [comment]
```
if isinstance(var_names, str):
    var_names = [var_names]
```

I have used this in io files.

**Comment by canyon289** [comment]
We should make this a central function so we can alter input behavior more easily later

**Comment by canyon289** [comment]
What are you referring to when you say manually fixing this?

I was referring to this
```
def get_var_names(var_names):
    if isinstance(var_names, str):
        var_names = [var_names]
    return var_names
```

**Comment by ahartikainen** [comment]
That seems easier to handle and test. I will remove the current PR.

**Comment by canyon289** [comment]
@ahartikainen Do you mind if I pick this one up?

**Comment by ahartikainen** [comment]
Yes, you can take it :) notice that some plots need atleast two variables, but even then I think function should fail as normally with a list of one var.

I'm not sure if we should update our docstrings.
